benchmarks/benchmark4.ipynb

```diff
@@ -134,7 +134,7 @@
    "source": [
```

Comment:
<p>The equation failing to render needs escapes on the <code>{</code>,<code>}</code> delimiters:</p><p> </p><pre class="ql-syntax" spellcheck="false">\begin{equation}
\frac{\partial\eta}{\partial t} = \nabla\cdot\left[M\nabla \left\{ \frac{\delta \mathcal{F}}{\delta\eta} \right\} \right],
\end{equation}
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/usnistgov/pfhub/pull/1485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
It isn't failing to render for me or on the [website](https://random-cat-1485.surge.sh/benchmarks/benchmark4.ipynb/). Where are you seeing this?

Reply:
Ah, you're right! It doesn't render on ReviewNB, but it looks fine on the website. Must be different math renderers.

Reply:
Oh, it's failing on ReviewNB only.

Reply:
<p>Also, the escapes are actually in the equation in the notebook. I think you must have added them some time ago.</p>